documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -76,10 +74,12 @@ export async function POST({ request, platform }) {
 
 > SvelteKit's built-in `$env` module should be preferred for environment variables.
 
-To make these types available to your app, reference them in your `src/app.d.ts`:
+To make these types available to your app, install `@cloudflare/workers-types` then reference them in your `src/app.d.ts`:
```

Comment:
`@cloudflare/workers-types` is a dependency of `adapter-cloudflare`, so I don't think you should need to install them separately?

Reply:
Let's remove this for now :pray: